web/packages/teleterm/src/main.ts

```diff
@@ -97,16 +90,17 @@ function initializeApp(): void {
 
   app.on('will-quit', async event => {
     event.preventDefault();
+    const disposeMainProcess = async () => {
```

Comment:
I would put it as a top-level function, but it would require passing two arguments to it :/ I guess this is better?

Reply:
That's fine I guess, it's only called once. I suppose using a promise also would be a good enough solution here.